src/sgame/sg_momentum.cpp

```diff
@@ -353,9 +353,7 @@ void G_DecreaseMomentum()
  */
 float G_AddMomentumGeneric( team_t team, float amount )
```

Comment:
These could just return void, it seems.

Reply:
It could yeah. I've kept the return value because it could be useful, but this is true.